optuna_dashboard/ts/components/StudyList.tsx

```diff
@@ -126,6 +126,71 @@ export const StudyList: FC<{
 
   const toolbar = <HomeIcon sx={{ margin: theme.spacing(0, 1) }} />
 
+  let studyListContent
+  if (isLoading) {
+    studyListContent = (
+      <div style={{ margin: theme.spacing(2) }}>
+        <SvgIcon fontSize="small" color="action">
+          <HourglassTop />
+        </SvgIcon>
+        Loading studies...
+      </div>
```

Comment:
[nit] In MUI, it is more common to use the `Box` component with `sx` properties.
https://mui.com/system/getting-started/the-sx-prop/

```suggestion
      <Box sx={{ margin: theme.spacing(2) }}>
        <SvgIcon fontSize="small" color="action">
          <HourglassTop />
        </SvgIcon>
        Loading studies...
      </Box>
```

Reply:
Thank you for your feedback.